### DISCUSSION TITLE
Start nexrender-server with HTTPS instead of HTTP

### DISCUSSION THREAD

**Discussion by joepenjoyce** [inlife/nexrender#855] [Ideas]
Is it possible to start the nexrender-server with HTTPS?
When I use fetch() on a webpage which is served over HTTPS and call the HTTP API I get security errors in Google Chrome.

**Comment by JamesBotterill** [comment]
I wouldn't be interacting with the server directly from any webpage.

You'll expose the server secret and would enable anyone to render anything from your instance and potentially cause licence issues for you.

I would set up a small node app that sits in-front of your webpage that interacts as the to the server. Then have a api that your website uses to handle your interactions with the server.
